lib/build.gradle

```diff
@@ -15,7 +15,7 @@ plugins {
     id("maven-publish")
 }
 
-version = '0.1.1'
+version = '0.0.11'
```

Comment:
Why this version number? We should only increment upwards, perhaps 0.2.0 would be better.

Reply:
Right you are...